kola/harness.go

```diff
@@ -151,8 +164,20 @@ func runTest(t *Test, pltfrm string) error {
 	for k := range t.NativeFuncs {
 		names = append(names, k)
 	}
+
+	// prevent unsafe access if tests ever become parallel and access
+	tempTestOptions := make(map[string]string, 0)
```

Comment:
Yeah, I figured it was unlikely, but just being careful. It would be odd for a test to do any writes, but, the api doesn't prevent it either I guess.